includes/class-wc-siftscience-comm.php

```diff
@@ -19,21 +19,21 @@ class WC_SiftScience_Comm {
 		private const DELETE_URL = 'https://api.sift.com/v204/users/{user}/labels/?api_key={api}&abuse_type=payment_abuse';
 		private const SCORE_URL = 'https://api.sift.com/v204/score/{user}/?api_key={api}';
 
-		private $options;
-		private $logger;
+		private $_options;
+		private $_logger;
 
 		private $headers = array(
```

Comment:
I guess this should be changed with underscore or even const if it's possible for arrays.

Reply:
didn't c that